runtime/doc/usr_27.txt

```diff
@@ -83,7 +83,7 @@ matter if 'ignorecase' or 'smartcase' was changed.
 
 	Note:
 	If your search takes much longer than you expected, you can interrupt
-	it with CTRL-C on Unix and  CTRL-Break on MS-DOS and MS-Windows.
+	it with CTRL-C on Unix and CTRL-Break on Windows.
```

Comment:
[This page](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682541%28v=vs.85%29.aspx) seems to agree with you. Do you have a system with Windows that this can be tested on? If Neovim doesn't work on Windows yet, then I bet Vim could still be used to demonstrate `CTRL-C` behavior in console.